integration_tests/mockApis/auth.ts

```diff
@@ -19,10 +19,12 @@ const createToken = () => {
 }
 
 const getLoginUrl = () =>
-  getRequests().then((data) => {
+  getMatchingRequests({
```

Comment:
this is a much nicer way to get the requests out of wiremock - retrieving all the ones matching a specific url rather than all the requests that exist